crates/runc-shim/src/task.rs

```diff
@@ -203,7 +203,7 @@ where
         resp.pid = pid;
 
         containers.insert(id.to_string(), container);
-
+        drop(containers);
```

Comment:
I'm pretty sure most of these statements can be rewritten in the chain form like this:

```rust
        self.containers
            .write()
            .await
            .insert(id.to_string(), container);
```

so we don't have to manually call drop everywhere.